dev/service-registry/client-core/src/test/scala/com/lightbend/lagom/devmode/internal/registry/LagomDevModeServiceDiscoverySpec.scala

```diff
@@ -11,6 +11,7 @@ import akka.actor.ActorSystem
 import akka.discovery.ServiceDiscovery.Resolved
 import akka.discovery.ServiceDiscovery.ResolvedTarget
 import akka.testkit.TestKit
+import com.lightbend.lagom.internal.registry.ServiceRegistryClient
```

Comment:
This is surprising. `ServiceRegistryClient` should remain on the same package than `LagomDevModeServiceDiscovery` so I suspect the package of `ServiceRegistryClient` needs renaming too.

Reply:
It will be fixed